invisible_cities/reco/krmap_functions_test.py

```diff
@@ -104,3 +105,62 @@ def test_transform_parameters(a, b):
     npt.assert_allclose(transformed_par, [  E0_expected,   lt_expected])
     npt.assert_allclose(transformed_err, [s_E0_expected, s_lt_expected])
     assert np.isclose(transformed_cov, cov_expected)
+
+
+@given(integers(min_value = 0, max_value = 1e10),
+       integers(min_value = 0, max_value = 1e10),
+       arrays  (dtype     = np.int64, shape = (2,),
+                elements  = integers(min_value = 1,
+                                     max_value = 1e4)))
+def test_get_number_of_bins_performance_default_value(nevents, thr, n_bins):
+    bins = krf.get_number_of_bins(nevents, thr, n_bins)
+
+    npt.assert_array_equal(bins, n_bins)
+
+
+@given(integers(min_value = 0, max_value = 1e10),
+       integers(min_value = 0, max_value = 1e10))
+def test_get_number_of_bins_with_thresholds(nevents, thr):
+    bins = krf.get_number_of_bins(nevents, thr)
+
+    if nevents < thr:
+        npt.assert_array_equal(bins, np.array([50, 50]))
+    if nevents >= thr:
+        npt.assert_array_equal(bins, np.array([100, 100]))
+
+
+@given(integers(min_value = 0, max_value = 1e10),
+       integers(min_value = 0, max_value = 1e10),
+       arrays  (dtype = np.int64,  shape = (2,),
+                elements = integers(min_value = 1,
+                                    max_value = 1e4)))
+def test_get_number_of_bins_returns_type(nevents, thr, n_bins):
+
+    assert type(krf.get_number_of_bins(nevents, thr) ) == np.ndarray
+    assert type(krf.get_number_of_bins(n_bins=n_bins)) == np.ndarray
```

Comment:
Remove hypothesis and do a simple fixed cases. Why do we want to test the output type?